src/stanza/fortex/stanza/stanza_processor.py

```diff
@@ -46,19 +46,21 @@ def initialize(self, resources: Resources, configs: Config):
             "pos" in configs.processors
             or "lemma" in configs.processors
             or "depparse" in configs.processors
+            # or "ner" in configs.processors
+            or "ner" in configs.processors
         ):
             if "tokenize" not in configs.processors:
                 raise ProcessorConfigError(
                     "tokenize is necessary in "
                     "configs.processors for "
-                    "pos or lemma or depparse"
+                    "pos or lemma or depparse or ner"
                 )
         self.set_up()
         self.nlp = stanza.Pipeline(  # type: ignore
             lang=self.configs.lang,
             dir=self.configs.dir,
             use_gpu=self.configs.use_gpu,
-            processors=self.configs.processors,
+            processors=self.configs.processors.todict(),
```

Comment:
Again, we can just use `self.processors` here right? Since it is being set when `self.set_up()` is called in L58.